qutebrowser/mainwindow/tabwidget.py

```diff
@@ -819,15 +820,15 @@ def _get_icon_rect(self, opt, text_rect):
                       else QIcon.Off)
         # reserve space for favicon when tab bar is vertical (issue #1968)
         position = config.val.tabs.position
-        if (opt.icon.isNull() and
-                position in [QTabWidget.East, QTabWidget.West] and
+        if (position in [QTabWidget.East, QTabWidget.West] and
```

Comment:
I think that this will cause a blank space to be displayed for sites without an icon (which would be annoying for horizontal tabs). I feel like this should either only take effect in vertical tabs, or it should be a setting somehow. I'm not sure about this though, it needs more thought.

Reply:
@jgkamat: This will only occur with vertical tabs, horizontal tabs should be completely unaffected by this change (apart from centering the icon in the available space).

This line in particular only matches vertical tabs:

```
position in [QTabWidget.East, QTabWidget.West]
```

Reply:
Ah, I probably should test things before assuming what the code does, you're right! :)

Reply:
No worries! Just pushed an updated based on your other comment.